dev-requirements.txt

```diff
@@ -1,2 +1 @@
--r requirements.txt
-http://downloads.sourceforge.net/project/pyasn1/pyasn1/0.1.8/pyasn1-0.1.8rc2.tar.gz?r=http%3A%2F%2Fsourceforge.net%2Fprojects%2Fpyasn1%2Ffiles%2Fpyasn1%2F0.1.8%2F&ts=1426346728&use_mirror=heanet

```

Comment:
pyasn1 is required for running tests, but not necessarily for installing (we can generate pyasn1 code without pyasn1 being installed.) Is there an 0.1.8 release on PyPI now, so we can just say `pyasn1>=0.1.8`